docs/data/remote-examples/templates/app.yaml

```diff
@@ -20,4 +20,5 @@ repos:
     - wordpress-composer
     - wordpress-vanilla
     - quarkus
+    - spring
```

Comment:
This is failing since we don't have a plain `spring` template: https://github.com/platformsh-templates?q=spring&type=&language=

Reply:
hum, I got it.